wrappers/windres-wrapper.sh

```diff
@@ -1,3 +1,183 @@
-#!/bin/sh
-set -e
-x86_64-w64-mingw32-windresreal "$@" --output-format res
+#!/bin/bash -e
+#
+# author: Josh de Kock <josh@itanimul.li>
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# For more information, please refer to <http://unlicense.org/>
+
+PROG="$0"
+
+CC=${CC:-i686-w64-mingw32-clang}
+
+print_version () {
+    cat <<EOF >&2
+version: ?xd
```

Comment:
I’m not entirely sure what version to put here, since some programs will probably expect some output here (they’ll parse this or something)

Reply:
I don't think I've see anything try to parse the windres version output (although since I've used the real windres, I have always had proper output here so I wouldn't have noticed either).